docs/farming-&-staking/farming/additional-guides/network-info.md

```diff
@@ -6,72 +6,117 @@ slug: networking
-| 30433 | TCP      | Always public                                           | Autonomys networking P2P    | <ul><li>[Space Acres](../../../category/space-acres-recommended/)</li><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli)</li></ul> | Connecting to the rest of the network and retrieving archival history.            |
-| 30533 | TCP     | Always public                                           | Autonomys networking P2P    | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli)</li></ul>                                                                        | Connecting to the rest of the network and retrieving archival history, not required for Space Acres because the node and the farmer share the Autonomys networking stack there. |
-| 40333 | TCP          | Always public                                           | Autonomys networking P2P    | <ul><li>[Advanced CLI Node (Domain)](../../../category/operators-and-nominators)</li></ul>                                                                           | Connecting to the rest of the domain network for transaction gossipping.      |
-| 9944  | TCP (HTTP/WebSocket)| Public: <ul><li>RPC server</li></ul> Private: <ul><li>Internal node-to-farmer connections</li></ul> | Consensus RPC port         | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli)</li></ul>                                                                        | RPC server: Exposes various details about the blockchain (real-time and historical events, storage, transactions, etc) as well as submitting transactions to the network (balance transfers, staking, etc). <br /><br /> Farmer connections: The farmer connects in unsafe mode and receives notifications about slot challenges for block/vote production, archival history extension events, querying of various chain details. Nodes that process farmer connections should never be exposed publicly as public exposure of unsafe APIs can be used to not only prevent farmer from working, but even block node sync completely. |
-| 9945  | TCP (HTTP/WebSocket)| Public: <ul><li>RPC server</li></ul>                     | Domain RPC port            | <ul><li>[Advanced CLI Node (Domain)](../../../category/operators-and-nominators)</li></ul>                                                                           | RPC server: Expose various details about blockchain (real-time and historical events, storage, transactions, etc.) as well as submitting transactions to the network (balance transfers, interacting with smart contracts, etc). |
+| Port | Protocol | Component| Purpose |
+|--|--|--|--|
+| 30333  | TCP | Node (Consensus) | Facilitate block, transaction, and PoT gossip exchange. |
+| 30433  | TCP | Farmer | Retrieve archival history from the blockchain. |
+| 30533  | TCP | Farmer | Synchronize with node peers to access archival history. |
```

Comment:
I believe we need to add port 9944 TCP, but @randywessels please double-check me on this

Reply:
I based this off of a [message](https://discord.com/channels/864285291518361610/1062507270539321485/1211854604195598396) from Nazar that states port 9944 should not be forwarded.

Reply:
Agree, it should not be forwarded, however, is it used for node <> farmer communication. 
I guess it depends on whether we want to show all used ports, or the ports that needs to be forwarded only

Reply:
Firewall instructions for port 9944 are currently covered in the `Linux Firewall Configuration` section, but it might be more appropriate to move this information into a dedicated `Firewall Rules` guide. Having firewall instructions in a Port Forwarding guide feels out of place. A separate guide would also allow us to include firewall rule instructions for Windows and other operating systems.

Reply:
I think 9944 is only needed if you are running a node that is not on the same box as your farmer or you are using docker containers.  But yeah it wouldn't be forwarded, just opened on the firewall.